util/stringutil/string_util_test.go

```diff
@@ -181,6 +181,40 @@ func (s *testStringUtilSuite) TestIsExactMatch(c *C) {
 	}
 }
 
+func (s *testStringUtilSuite) TestBuildStringFromLabels(c *C) {
+	defer testleak.AfterTest(c)()
+	testcases := []struct {
+		name     string
+		labels   map[string]string
+		expected string
+	}{
+		{
+			name:     "nil map",
+			labels:   nil,
+			expected: "",
+		},
+		{
+			name: "one label",
+			labels: map[string]string{
+				"aaa": "bbb",
+			},
+			expected: "aaa=bbb",
+		},
+		{
+			name: "two labels",
+			labels: map[string]string{
```

Comment:
Could you fix the test? #19305

Reply:
The error has already been fixed in this issue. @ekexium